### ISSUE TITLE
Update for NServiceBus6

### ISSUE THREAD

**Issue by bhardwajgaurav007** [benjaminramey/GoodlyFere.NServiceBus.EntityFramework#3]
Hi,

Are there any plans of updating this component to make it compatible with NServiceBus 6? 

Thanks,

**Comment by bhardwajgaurav007** [comment]
Hi, 
Just a gentle reminder!
Thanks,

**Comment by benjaminramey** [comment]
Thanks for the reminder and sorry I was slow to respond!  

I'll be frank and say, no there are no plans to upgrade this library further.  Interest from folks around the NServiceBus community has been meager at best.  I even ended up removing it from the project I built it for because I was running into weird issues that I had no time to investigate and fix.

I switched to the Particular-provided NHibernate solution and all my issues went away.  So, I didn't have much incentive to keep this library up-to-date.  :-)

What's your situation?  Are you already using this library and you want to upgrade to NServiceBus 6?  Or are you just looking for EntityFramework persisters for a new installation?

**Comment by bhardwajgaurav007** [comment]
Yes, we are using this library and now want to upgrade to NServiceBus6. We are using PostgreSQL with EF6 in our solution and wanted to use same for NSeviceBus persistence also and that is the reason we have not used NHibernate for NServiceBus.

**Comment by benjaminramey** [comment]
OK, I'm glad to hear of someone else using it!

I can see if I have time to look into what it would take to upgrade to NSB6 some time this week.  I don't want to leave you hanging.

**Comment by sonaljain14** [comment]
I also plan to use this library. Will be great if there is support for new version of nservicebus

**Comment by SimonCropp** [comment]
Not to say this is a better option to the awesome work done by Ben. But this might be a viable alternative. https://docs.particular.net/nservicebus/sql-persistence/  

Either way I would appreciate any feedback

**Comment by sonaljain14** [comment]
We are using postgresql, and that is the reason we wanted to go with Ben's work.

**Comment by bhardwajgaurav007** [comment]
Did you get a chance to work on this?

Thanks,

**Comment by weralabaj** [comment]
We're tracking interest in Postgres persistence and will post updates in this issue https://github.com/Particular/NServiceBus.Persistence.Sql/issues/15. Make sure to comment and follow if you're interested, thanks!
